proto/proto/store/rpc.proto

```diff
@@ -186,7 +186,9 @@ message CheckNullifiersByPrefixRequest {
     // to `prefix_len`.
     repeated uint32 nullifiers = 2;
     // Block number from which the nullifiers are requested (inclusive).
-    fixed32 block_num = 3;
+    fixed32 block_from = 3;
+    // Block number up to which to check. If not specified, checks up to the latest block.
+    optional fixed32 block_to = 4;
```

Comment:
I think we should adapt a convention where all "sync" requests specify block from/to as the first two fields. For this message this would look something like:
```proto
message CheckNullifiersByPrefixRequest {
    fixed32 block_from = 1;
    optional fixed32 block_to = 2;
    uint32 prefix_len = 3;
    repeated uint32 nullifiers = 3;
}
```
This is already the case for `SyncStorageMapsRequest` and `SyncAccountVaultRequest`, and we should make it the same for `SyncNotesRequest` (in a different PR - let's make an issue for this).